modules/hello-world/pages/platform-help.adoc

```diff
@@ -0,0 +1,172 @@
+= Setting Up Couchbase PHP SDK with VSCode
+:description: pass:q[Discover how to get up and running developing applications with the Couchbase PHP SDK 3.0+ using `Visual Studio Code`.]
+:page-aliases: ROOT:platform-introduction,ROOT:platform-help,php-environment
```

Comment:
Not sure if these have all been used, a quick look at https://docs-archive.couchbase.com/php-sdk/2.1/overview.html isn't turning up anything

Reply:
copy/pasta from the .NET one... will remove